.github/workflows/ubi-ppc64le-build.yml

```
@@ -0,0 +1,50 @@
+name: UBI Image Builder
+
+on:
+  workflow_dispatch: 
+  push: 
+    paths:
+      - 'ubi-ppc64le/**'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: 'ubi-ppc64le'
```

this is not just enough, 
```suggestion
  IMAGE_NAME: ${{ github.repository_owner }}/'ubi-ppc64le'
```

so that it will get published to `ghcr.io/osuosl/ubi-ppc64le`

---

Thanks, I missed it.
I have updated the workflow with the owner